content/en/tracing/llm_observability/auto_instrumentation.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Auto Instrumentation
+---
+
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+Datadog's [LLM Observability Python SDK][16] provides integrations that automatically trace and annotate calls to LLM frameworks and libraries. Without changing your code, you can get out-of-the-box traces and observability for calls that your LLM application makes to the following frameworks:
+
+
+| Framework                               | Supported Versions |
+|-----------------------------------------|--------------------|
+| [OpenAI](#openai)                       | >= 0.26.5          |
+| [Langchain](#langchain)                 | >= 0.0.192,<0.2.0  |
+| [AWS Bedrock](#aws-bedrock)             | >= 1.31.57         |
+| [Anthropic](#anthropic)                 | >= 0.28.0          |
+
+In addition to capturing latency and errors, the integrations capture the input parameters, input and output messages, and token usage (when available) of each traced call.
+
+## Enabling and disabling integrations
+
+All integrations are enabled by default.
+
+To disable all integrations, use the [in-code SDK setup][12] and specify `integrations_enabled=False`.
+
+To only enable specific integrations:
+1. Use the [in-code SDK setup][12], specifying `integrations_enabled=False`.
+2. Manually enable the integration with `ddtrace.patch()` at the top of the entrypoint file of your LLM application:
+
+{{< code-block lang="python">}}
+from ddtrace import patch
+from ddtrace.llmobs import LLMObs
+
+LLMObs.enable(integrations_enabled=False, ...)
+patch(<INTEGRATION_NAME_IN_LOWERCASE>=True)
+{{< /code-block >}}
+
+**Note**: Use `botocore` as the name of the [AWS Bedrock](#aws-bedrock) integration when manually enabling.
+
+## OpenAI
+
+The OpenAI integration provides automatic tracing for the [OpenAI Python SDK's][1] completion and chat completion endpoints.
+
+### Traced methods
+
+The OpenAI integration instruments the synchronous, asynchronous, and streamed versions of the following methods:
+
+- [Completions][2]:
+   - `OpenAI().completions.create()`
+   - `AsyncOpenAI().completions.create()`
+- [Chat completions][3]:
+   - `OpenAI().chat.completions.create()`
+   - `AsyncOpenAI().chat.completions.create()`
+
+## LangChain
+
+The LangChain integration provides automatic tracing for the [LangChain Python SDK's][4] LLM, chat model, and chain calls.
+
+### Traced methods
+
+The LangChain integration instruments both synchronous and asynchronous versions of the following methods:
+
+- [LLMs][13]: 
+  - `llm.invoke()`
+  - `llm.batch()`
+- [Chat models][14] 
+  - `chat_model.invoke()`
+  - `chat_model.invoke()`
```

Comment:
@Yun-Kim , did you mean something else for one of these?

Reply:
Yes! Let me make an edit to fix this, thanks for the catch 😄